core/src/main/java/org/mskcc/cbio/oncokb/util/MainUtils.java

```diff
@@ -512,4 +512,13 @@ public static Boolean isVUS(Alteration alteration) {
         List<Evidence> evidenceList = EvidenceUtils.getEvidence(Collections.singletonList(alteration), Collections.singleton(EvidenceType.VUS), null);
         return !(evidenceList == null || evidenceList.isEmpty());
     }
+
+    public static boolean isInteger(String str) {
```

Comment:
I think here you just need to know whether str is number? You can use `isNumber` method in apache common package